akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -536,6 +536,19 @@ object TestSubscriber {
       }
     }
 
+    /**
+     * Expect next element and test it with partial function.
+     *
+     * Allows chaining probe methods.
+     */
+    def expectNextChainingPF(f: PartialFunction[Any, Any]): Self = {
+      expectEventPF {
+        case OnNext(n) ⇒
+          assert(f.isDefinedAt(n))
```

Comment:
Hmmm... Just checked it and got same message for expectNextPF, assert() call in 547 throws AssertionError with "assertion failed" message if PF fails to match, same is in expectNextPF.